msi/GitProduct.wxs

```diff
@@ -38,7 +38,7 @@
 
     <!-- LINEENDINGS Property implementing the "Remember Property" pattern -->
     <Property Id="LINEENDINGS" Value="true">
-      <RegistrySearch Id="LineEndingsSearch" Root="HKLM" Key="SOFTWARE\GitForWindows" Name="LineEndings" Type="raw" />
+      <RegistrySearch Id="LineEndingsSearch" Root="HKLM" Key="SOFTWARE\GitForWindows" Name="core.autocrlf" Type="raw" />
```

Comment:
Can this change go back into the commit that added the "Remember Property" patterns?